packages/pages/src/dev/server/middleware/serverRenderRoute.ts

```diff
@@ -88,9 +88,12 @@ export const serverRenderRoute =
         return;
       }
 
+      const overrides = JSON.parse(req.body.overrides);
```

Comment:
Is this safe if there are no overrides or body?

Reply:
Updated, should be safe now